src/pages/preferences/preferencesTabs/cliTab/nodeList/Node.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import Styled from "@emotion/styled";
+import { Subtitle } from "@leafygreen-ui/typography";
+import { uiColors } from "@leafygreen-ui/palette";
+
+const { green, white } = uiColors;
+
+export type NodeType = {
+  title: string;
+  child?: JSX.Element;
+};
+
+interface NodeProps extends NodeType {
+  index: number;
+}
+export const Node: React.FC<NodeProps> = ({ title, child, index }) => (
+  <NodeContainer>
+    <NodeHeader>
+      <Step index={index + 1} />
```

Comment:
I think it makes more sense for Node to accept a stepNumber prop from NodeList instead of index because the looping logic happens in NodeList and Node doesn't need to know that it's indexed starting at 0